+ 24
- 0
Viewed
From 6d5dc2943033bde34ecf50ede88e8be551a0dbd8 Mon Sep 17 00:00:00 2001
From: Alexandre FISET <alexandre.fiset_ext@softathome.com>
Date: Mon, 24 Oct 2022 12:53:19 +0200
Subject: [PATCH] tr181-dnssd: mDNS service is not available
 over IPv6

GitOrigin-RevId: af2459184c76fb3cafc3147ac684ff2713efa163

diff --git a/src/firewall_utility.c b/src/firewall_utility.c
index cf2628e..e87bf7f 100644
--- a/src/firewall_utility.c
+++ b/src/firewall_utility.c
@@ -96,7 +96,7 @@ void firewall_open_dnssd_port(intf_info_t* info) {
     amxc_var_add_key(uint32_t, &args, "destination_port", DNSSD_PORT);
     amxc_var_add_key(bool, &args, "enable", true);
     amxc_var_add_key(cstring_t, &args, "interface", info->intf_path);
-    amxc_var_add_key(uint32_t, &args, "ipversion", 4);
+    amxc_var_add_key(uint32_t, &args, "ipversion", 0);
     amxc_var_add_key(cstring_t, &args, "protocol", "UDP");
 
     controller = amxd_object_get_value(cstring_t, info->dnssd, "FWController", NULL);
-- 
GitLab
